Prox/Prox/Utilities/OpenInHelper.swift

```diff
@@ -33,6 +34,15 @@ struct OpenInHelper {
         return true
     }
 
+    private static func extractHostname(fromHost host: String) -> String? {
+        let components = host.components(separatedBy: ".")
+        if components.count > 0 {
```

Comment:
I think I'll keep it as is, though I'll document the assumption about the format of the url.